ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -769,6 +839,49 @@
                       default-excludes: true
                       fingerprint: false
                       only-if-success: false
+      - '{name}-{test_name}-for-period':
+          test_name: windows-e2e-userspace
+          node: 'antrea-windows-testbed'
+          description: 'This is for running e2e tests with OVS userspace daemons running as Windows services.'
```

Comment:
I do more like `windows-e2e-with-ovs` to present that OVS is running with agent together inside a Pod, because "userspace" is not clear for me to know that OVS is expected to run on the Windows host as a native service. I think we shall get an agreement that the case name shows what modules are started (run) in the job. If so, we shall include "ovs" if we will use the manifest to run OVS (with container mode), and omit when OVS is already running on the host (with service mode).

Reply:
okay